tools/generators/lib/PBXProj/src/Platform.swift

```diff
@@ -1,6 +1,13 @@
 import ArgumentParser
 
 public enum Platform: String, ExpressibleByArgument {
+    public enum OS: Comparable {
```

Comment:
Just curious, is visionOS going to be a separate platform like this? Haven't looked into anything about it yet lol

Reply:
Yes